src/recipes/recipes.repository.ts

```diff
@@ -111,4 +114,17 @@ export class RecipesRepository {
 
     return { recipes: recipes };
   }
+
+  async getRecipesById(id: string): Promise<GetRecipeResponse> {
+    return await this.prisma.recipe.findUnique({
```

Comment:
You probably don't need `return await` here as this is the only method here. Just `return` should be enough (then remove `async` keyword; `Promise< GetRecipeResponse>` return type should stay).